app-shells/zsh-theme-powerlevel9k/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add proxy maintainers project last, as you are not a gentoo developer

Reply:
Sorry, but i just don't understand this problem...(i'm french sorry).